### ISSUE TITLE
QMK Like Magic Swap Functions

### ISSUE THREAD

**Issue by anthonygualandri** [tmk/tmk_keyboard#757]
Is there a way to do QMK like magic swap functions for reversing the Lctrl and Lgui (LCG_SWP) for changing between Windows and Mac machines?

**Comment by tmk** [comment]
This perhaps?
https://github.com/tmk/tmk_keyboard/wiki/TMK-Magic-Command#keymap

**Comment by anthonygualandri** [comment]
Yeah. That’s it. And is that possible using the TMK online editor? If so, how?On Apr 8, 2023, at 8:52 PM, ***@***.*** wrote:﻿
This perhaps?
https://github.com/tmk/tmk_keyboard/wiki/TMK-Magic-Command#keymap

**Comment by tmk** [comment]
Probably no.

It depends on product. The feature is not enabled by default except for Alps64.
You will have to build firmware yourself in command line.
